src/RuleSet/DeclarationBlock.php

```diff
@@ -181,15 +181,13 @@ public function getSelectors()
      * Splits shorthand declarations (e.g. `margin` or `font`) into their constituent parts.
      *
      * @return void
+     *
+     * @deprecated This will be removed without substitution in version 10.0.
```

Comment:
I think we should add the deprecated marker now but not change anything about the functionality until we hit version 10.

Reply:
I'm not sure, because I was just following the suggestions from https://github.com/MyIntervals/PHP-CSS-Parser/issues/512#issue-2162927498 where @oliverklee mentioned to

> [...] remove the functionality, make all the corresponding methods a no-op and mark them as @deprecated [...].

Is this still the way to go, or do we want to just put a `@deprecated` on the function? Also, shall we emit a warning?

Reply:
I'd say we only mark it as `@deprecated`. (I honestly don't remember why I tended to make it a no-op in #512). This will allow people to adapt their code when they see that their code is calling a deprecated method.

Reply:
So no `trigger_error()`?

Reply:
Yes, exactly. (In the future, we're going to also have a separate test suite for deprecated code that will allow deprecation warnings. Then we'll also be able to use `trigger_error`.)

Reply:
I marked it as deprecated and (partly) reverted the commits.